internal/pkg/model/object.go

```diff
@@ -142,6 +142,11 @@ type Code struct {
 	Scripts      []string `json:"script"` // scripts, eg. SQL statements
 }
 
+// Schedule - https://app.swaggerhub.com/apis/odinuv/scheduler/1.0.0#/schedules/get_schedules
+type Schedule struct {
+	Id string `json:"id" validate:"required" diff:"true" metaFile:"true"`
```

Comment:
`diff:"true" metaFile:"true"` su tu zbytocne, nikde sa nepouzivaju ... ak budu potrebne, tak sa pridaju neskor

Reply:
Jo tak to jsem vlastně úplně slepě opsal a ani nekoukal co to. vlastně znamená. 🙈 Jdu to dostudovat. 😅